Update targetting-user readme (#213)
launchdarkly/LaunchDarkly-Docs: src/content/topics/home/using-flags/targeting-users.mdx

You can add multiple conditions to a rule. Users must meet all the conditions in a rule to match the rule. If any of the conditions are not met, the user will not match the rule. If a custom attribute has an array value, LaunchDarkly treats it as multiple values and allows any of the values within the array to match a rule.

> Here, we've created a custom rule with two conditions. This rule serves `true` to all "Not test users" who are not in "Test Segment" and their `account` attribute does not start with `test`.

Reviewer: ```suggestion
Here, we've created a custom rule with two conditions. This rule serves `true` to users who are not in the `Test Segment` and whose `Account` attribute does not start with `test`.
```
